testing/oauth_server.ts

```diff
@@ -67,6 +67,7 @@ export function launchOAuthServerFlow({
     client_id: clientId,
     redirect_uri: redirectUri,
     response_type: 'code',
+    state: new Date().getTime(),
```

Comment:
Mind adding a note that this is here because some providers want this parameter but for CLI purposes it doesn't matter what's in it?